README.md

```diff
@@ -44,7 +44,7 @@ flowchart LR
 
 **3. Edit:**
 
-- On the top right of the Readme file, click on the pencil icon to edit the file by adding your name.
+- On the top right of the Readme file, click on the pencil icon to edit the file by **adding your name to the section that matches your Initial in [this list](https://github.com/EddieHubCommunity/hacktoberfest-practice#hacktoberfest-community), make sure that your name is in alphabetical order.**
```

Comment:
```suggestion
- On the top right of the Readme file, click on the pencil icon to edit the file by **adding your name to the section that matches your Initial in [this list](https://github.com/EddieHubCommunity/hacktoberfest-practice#hacktoberfest-community). Make sure that your name is in alphabetical order.**
```

Reply:
Thank you @Panquesito7 
I have applied those changes😊